Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_001.xml

```diff
@@ -0,0 +1,374 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
On 4/22/16 8:10 PM, Leszek Grzanka wrote:
> In
> Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_001.xml
> <https://github.com/cms-sw/cmssw/pull/14029#discussion_r60780079>:
> 
>> @@ -0,0 +1,374 @@
>> +<?xml version="1.0" encoding="UTF-8"?>
> 
> Right now we use such arrangement to adjust position of the movable
> sensors via xml files containing constants with distances (i.e.
> Geometry/VeryForwardData/data/2015_10_18_fill4511/RP_Dist_Beam_Cent.xml).
> These files were used to adjust detector placement without the need to
> read it from database.
> 
> Constants from RP_Dist_Beam_Cent.xml are used i.e. in
> Geometry/VeryForwardData/data/RP_220_Right_Station.xml to properly
> position RP_Box_* volumes.
> 
> We have 24 constants corresponding to 24 RP_Box_* volumes and also to 24
> RP_Detectors_Assembly_/.xml files (each describing different RP_Box_/
> volume).
> 
> Can you suggest better solution which could be quickly implemented now ?



If this adjustment is needed realistically only on the reconstruction side,
you can use alignment.
I suppose, in simulation as well the alignment constants can be applied
if it changes physics
(we typically don't move other detectors in simulation from their design
positions).


> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/14029/files/b34061440368ca2b0e9f654860d37dd754a23c72#r60780079>
>